### DISCUSSION TITLE
lc/999/

### DISCUSSION THREAD

**Discussion by giscus** [doocs/leetcode#2701] [Announcements]
# lc/999/

多种编程语言实现 LeetCode、《剑指 Offer（第 2 版）》、《程序员面试金典（第 6 版）》题解

https://leetcode.doocs.org/lc/999/

**Comment by thinkasany** [comment]
这题输入有点超长了，是不是可以换行

**Reply by yanglbme** [reply]
这是框架渲染的，不知道能不能改
